src/main/java/edu/iis/powp/adapter/LinePlotterAdapter.java

```diff
@@ -24,17 +25,22 @@ public void setPosition(int x, int y)
 	    @Override
 	    public void drawTo(int x, int y)
 	    {
-	    	ILine line = LineFactory.getBasicLine();
+	    	ILine line = lineMain;
 	    	line.setStartCoordinates(this.startX, this.startY);
 	        line.setEndCoordinates(x, y);
 	        setPosition(x,y);
 	        ApplicationWithDrawer.getDrawPanelController().drawLine(line);
 	    }
 
+	    // W metodzie toString() nie umieściłem opcji "Client plotter 3.0 - Dotted Line - Color" ze względu na brak metody .isSpecial() / .isBasic() w LineFactory.
 	    @Override
 	    public String toString()
 	    {
-	        return "Client plotter 2.0";
+	    	if(lineMain.isDotted()) {
```

Comment:
What about special line type? It would be better to 'inject' also a plotter name.